plugins/inputs/kafka_consumer/README.md

```diff
@@ -28,6 +28,10 @@ from the same topic in parallel.
   ## more about them here:
   ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md
   data_format = "influx"
+
+  ## Maximum length of a message to consume, in bytes (default 0/unlimited);
+  ## larger messages are dropped
+  max_message_len = 64 * 1024
```

Comment:
I commented back a bit on this, but in case it is lost in the "outdated comments".  You can't do this, I just wasn't paying attention to this being TOML in my original comment.